### PR TITLE
fix: Squashed commits of various fixes #8408, #8332, #8354, #6525, #3372

### LINKED ISSUE #8408: Add a link to the Tutorials in the menu of the Pro platform #8407
We are starting a series of video tutorials to help producers navigate the Pro Platform.
It would be great to add a link in the menu to promote them.

Text: could be "Video Tutorials"

Link (same for all langages): https://www.youtube.com/playlist?list=PLjAH-USadsF3xgSVfUUBS4we3XBr-1r55

### PR DESCRIPTION
**Pull request by aadak99**
*Squashed my commits together*

#8408
Added YouTube link to producers menu for tutorials. 
@alexgarel revised it and added a string so that the video tutorials could be translated. 
***The only issue I ran into was, the link translation string needs to be added to common.po and common.pot, but I do not have permission to assist with that. It would need to be added for entry "video_tutorials"

#8332 
Added producers link at the top nav bar.
238206863-ccc567ae-9e75-4403-a489-8f897b4b2c31

#8354 
Added correct link under Italian translation. 
f6d9d038765525ec6fa269747bc17ab1

#6525 
Made fix to typo aspect to aspects.
e719bdabad6f25de6cde2c0170bc159a

#3372 
Updated world string to not change to FR on main English page of openfoodfacts
6a3ae453810a40b4a166f7204daf846c

Fixes #8408, #8332, #8354, #6525, #3372

Sorry for duplicate entries, trying to clean things up a bit and get it organized. Should be better going forward!

### CODE REVIEW

**Comment by alexgarel** [comment]
Hi @aadak99, thanks for this PR. Normally you should also have modifications to po/common.pot and to po/common/en.po to add your translations strings but I don't see them in the PR.

**Review by aadak99 — COMMENTED** [review]
Added translatable string for video_tutorials

File: templates/web/common/site_layout.tt.html [hunk 1]
```diff
@@ -216,6 +217,7 @@
 					<ul class="search_and_links">
 						<li><a href="[% lang('menu_discover_link') %]" class="top-bar-links">[% lang('menu_discover') %]</a></li>
 						<li><a href="[% lang('menu_contribute_link') %]" class="top-bar-links">[% lang('menu_contribute') %]</a></li>
+						<li><a href="[% lang('footer_producers_link') %]" class="top-bar-links">[% lang('footer_producers') %]</a></li>
```
Comment (stephanegigandet) [inline templates/web/common/site_layout.tt.html:220]: I don't think we have room for this link at all resolutions / all languages. Maybe we could display it only on very large screens.

image

Comment (alexgarel) [reply templates/web/common/site_layout.tt.html:220]: @aadak99 do you think you can tackle this ?

It's about adding a class "show-for-xlarge-up" for example. See https://get.foundation/sites/docs-v5/components/visibility.html

Comment (aadak99) [reply templates/web/common/site_layout.tt.html:220]: @stephanegigandet  @alexgarel 
I should be able to handle this as it is just an addition of a class. I'll have it done by tonight EST!

**Review by stephanegigandet — APPROVED** [review]
Thank you!

File: templates/web/common/site_layout.tt.html [hunk 2]
```diff
@@ -64,6 +64,7 @@
 
 									<li><label>[% lang("resources") %]</label></li>
 									<li>
+										<li><a href="https://www.youtube.com/playlist?list=PLjAH-USadsF3xgSVfUUBS4we3XBr-1r55" target="_blank">lang("video_tutorials")</a></li>
```
Comment (stephanegigandet) [inline templates/web/common/site_layout.tt.html:67]: This part of the code was recently moved by @MonalikaPatnaik in another file: web/common/producers_resources_list.tt.html , so we should add this new list item there.

Comment (MonalikaPatnaik) [reply templates/web/common/site_layout.tt.html]: Done!

**Review by stephanegigandet — APPROVED** [review]
Thank you very much @aadak99 and @MonalikaPatnaik !

**Comment by aadak99** [comment]
> Thank you very much @aadak99 and @MonalikaPatnaik !

@alexgarel @MonalikaPatnaik @stephanegigandet 
I appreciate everyone's help and patience with me, now that I understand mostly the process I will be helping out where I can!

**Comment by stephanegigandet** [comment]
> I appreciate everyone's help and patience with me, now that I understand mostly the process I will be helping out where I can!

Thank you so much @aadak99! :)

**Pull request merged**